docs/netdata-agent/versions-and-platforms.md

```diff
@@ -0,0 +1,70 @@
+
+Netdata Agents are available in 2 versions:
+
+| Release Channel |               Release Frequency               |                 Support Policy & Features                 |             Support Duration             |                              Backwards Compatibility                              |
+|:---------------:|:---------------------------------------------:|:---------------------------------------------------------:|:----------------------------------------:|:---------------------------------------------------------------------------------:|
+|     Stable      | At most once per month, usually every 45 days | Receiving bug fixes and security updates between releases | Up to the 2nd stable release after them  |     Previous configuration semantics and data are supported by newer releases     |
+|     Nightly     |           Every night at 00:00 UTC            |               Latest pre-released features                | Up to the 2nd nightly release after them | Configuration and data of unreleased features may change between nightly releases |
+
+> "Support Duration" defines the time we consider the release as actively used by users in production systems, so that all features of Netdata should be working like the day they were released. However, after the latest release, previous releases stop receiving bug fixes and security updates. All users are advised to update to the latest release to get the latest bug fixes. 
+
+## Binary Distribution Packages
+
+Binary distribution packages are provided by Netdata, via CI integration, for the following platforms and architectures: 
+
+|        Platform         |        Platform Versions         |          Released Packages Architecture          |    Format    |
+|:-----------------------:|:--------------------------------:|:------------------------------------------------:|:------------:|
+|   Docker under Linux    |         19.03 and later          | `x86_64`, `i386`, `ARMv7`, `AArch64`, `POWER8+`  | docker image |
+|      Static Builds      |                -                 | `x86_64`, `ARMv6`, `ARMv7`, `AArch64`, `POWER8+` |   .gz.run    |
+|       Alma Linux        |             8.x, 9.x             |               `x86_64`, `AArch64`                |     RPM      |
+|      Amazon Linux       |             2, 2023              |               `x86_64`, `AArch64`                |     RPM      |
+|         Centos          |               7.x                |                     `x86_64`                     |     RPM      |
+|         Debian          |         10.x, 11.x, 12.x         |       `x86_64`, `i386`, `ARMv7`, `AArch64`       |     DEB      |
+|         Fedora          |            37, 38, 39            |               `x86_64`, `AArch64`                |     RPM      |
+|        OpenSUSE         | Leap 15.4, Leap 15.5, Tumbleweed |               `x86_64`, `AArch64`                |     RPM      |
```

Comment:
We do not currently support Leap 15.4 (it went EOL a few months ago), and we dont currently publish Tumbleweed packages.